xrpl/transaction/main.py

```diff
@@ -139,5 +144,49 @@ def autofill(transaction: Transaction, client: SyncClient) -> Transaction:
         main.autofill(
             transaction,
             client,
+            signers_count,
+        )
+    )
+
+
+def safe_sign_transaction_with_multisign(
+    transaction: Transaction,
+    wallet: Wallet,
+) -> str:
+    """
+    Signs a transaction with multisign locally, without trusting external rippled nodes.
```

Comment:
nit: Same change as above
```suggestion
    Signs a transaction to be used for multisigning.
```